workflow/proc_pipe/mriqc/run_mriqc.py

```diff
@@ -6,19 +6,25 @@
 
 parser.add_argument('--global_configs', type=str, help='path to global configs for a given mr_proc dataset')
```

Comment:
I think this should be `--global_config` to match the [fMRIPrep runscript](https://github.com/neurodatascience/mr_proc/blob/main/workflow/proc_pipe/fmriprep/run_fmriprep.py#L138), the [trackers](https://github.com/neurodatascience/mr_proc/blob/main/trackers/run_tracker.py#L101), [DICOM](https://github.com/neurodatascience/mr_proc/blob/main/workflow/dicom_org/run_dicom_org.py#L103) [conversion](https://github.com/neurodatascience/mr_proc/blob/main/workflow/bids_conv/run_bids_conv.py#L146) scripts, etc.

Reply:
changed